app/src/main/kotlin/com/intive/picover/profile/view/ProfileUpdateBottomSheet.kt

```diff
@@ -61,11 +60,11 @@ fun ProfileUpdateBottomSheet(
 				actions = {
 					IconButton(
 						onClick = {
-							updateName(userName)
+							updateUsername()
```

Comment:
I would improve the name to clearly indicate that we save the data under the hood.
Proposal `saveUserName`

Reply:
Done